web/styles/settings.css

```diff
@@ -532,6 +532,15 @@ input[type="checkbox"] {
     }
 }
 
+.realm_enter_sends_kbd kbd {
+    opacity: 0.9;
+    font-size: 11px;
+    position: relative;
+    bottom: 1px;
+    margin: 0 2px;
+    color: hsl(0deg 0% 50%);
+}
```

Comment:
Why does this need custom CSS? We have top-level styles of `kbd` in `app_components.css`, and one should at least document why this needs to override taht.